src/components/Tour.vue

```diff
@@ -44,11 +44,45 @@ export default {
 				},
 				'ide-workspace': {
 					title: 'User Workspace',
-					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services and files.'
+					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services, custom processes and files.'
 				},
 				'ide-viewer': {
 					title: 'Data Viewer',
-					content: 'This area you can use to preview collections or inspect the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+					content: 'This area is for previewing collections or inspecting the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+				},
+
+				'connect-url': {
+					title: 'URL input',
+					content: `Paste the URL of the server you wish to connect to into this field. If you don't know any URL, you may head over to <a href="https://hub.openeo.org/">openEO Hub</a> to discover available servers.`
+				},
+				'connect-history': {
+					title: 'History of URLs',
+					content: 'Clicking on this icon will open a window that displays all URLs that you have connected to in the past (provided your browser permits storing in Local Storage).'
+				},
+
+				'login-options': {
+					target: '#credentials > .tabsHeader',
+					title: 'Login Options',
+					content: `You have several ways of authenticating with the server. The standard method is OpenID Connect (OIDC). If supported, you will see a button for each available OIDC provider here. Click on one to open its login form.`
+				},
+				'login-internal': {
+					target: '#credentials > .tabsHeader > button[title=Internal]',
+					title: 'User/Password',
+					content: `Some servers support basic authentication with a simple username-password combination. This method is discouraged and generally only used for internal development purposes (hence the name of this button). We suggest to use OpenID Connect if you can.`
+				},
+				'login-without': {
+					target: '#credentials > .tabsHeader > button[title="No credentials"]',
+					title: 'Without login',
+					content: `It is possible to proceed without any authentication, however, usually this means that you won't able to actually submit calculations etc. You can think of it as an "discovery only" mode.`
+				},
+				'login-credentials': {
+					target: '.tabContent:not([style*="display: none"]) .input',  // select first visible input
```

Comment:
I think this needs to be a bit more fine-grained. Especially, we would want to explain the Client ID field if it is shown.

Reply:
I can try whether I can detect whether there is OIDC auth on the current server or not, but at the moment I wouldn't know how to do this in Tour.vue :thinking: Is this info available via some `this.` variable?

Can you provide me with an explanation you would like to see for Client ID? I don't know too much about it myself...

Reply:
That's probably an issue, indeed. I've posted this as a separate task for later: https://github.com/Open-EO/openeo-web-editor/issues/172#issuecomment-914315738